scripts/destroy_stage.ts

```diff
@@ -0,0 +1,234 @@
+import AWS from 'aws-sdk'
+
+AWS.config.update({
+    region: 'us-east-1',
+})
+
+const stackPrefixes = [
+    'app-web',
+    'app-api',
+    'ui-auth',
+    'uploads',
+    'postgres',
+    'storybook',
+    'infra-api',
+    'ui',
+]
+
+const stage = process.argv[2]
+
+const cf = new AWS.CloudFormation()
+const s3 = new AWS.S3()
+
+async function main() {
+    stackPrefixes.map(async (prefix) => {
+        const sn = `${prefix}-${stage}`
+        const clearBucketOutput = await clearServerlessDeployBucket(sn)
+        if (clearBucketOutput instanceof Error) {
+            console.log(clearBucketOutput)
+        }
+        const deleteStackOutput = await deleteStack(sn)
+        if (deleteStackOutput instanceof Error) {
+            console.log(deleteStackOutput)
```

Comment:
Should we be returning a non-zero exit code if something fails here?

Reply:
For sure, I'll add that.